src/main/cpp/subsystems/HatchIntake.cpp

```diff
@@ -160,6 +162,21 @@ void HatchIntake::SetArmValue() {
     kPID_PrimaryClosedLoop, kTimeout_10Millis);
 }
 
+void HatchIntake::SetAngleValue() {
+  encoderFwd = mPivot.GetSelectedSensorPosition(kPID_PrimaryClosedLoop);
+  encoderRev = mPivot.GetSelectedSensorPosition(kPID_PrimaryClosedLoop);
```

Comment:
`encoderFwd` and `encoderRev` will always be the same value. `encoderFwd` should be captured in a different command. 

The command that calls `GetArmValuesFwd()`can capture `encoderFwd`, then the command that calls `SetAngleValue()` can capture `encoderRev`. 

That is, don't capture `encoderFwd` here. Assume there's a previous value.